packages/emotion/src/index.ts

```diff
@@ -21,15 +21,10 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-export { jsx, css, keyframes, Global } from '@emotion/react'
+/// <reference types="@emotion/react/types/css-prop" />
+
+export * from '@emotion/react'
 
 export { EmotionThemeProvider } from './EmotionThemeProvider'
-export {
-  ThemeablePropValues,
-  ThemeablePropTypes,
-  makeThemeVars,
-  getShorthandPropValue,
-  mirrorShorthandCorners,
-  mirrorShorthandEdges
-} from './styleUtils'
-export { withStyle } from './withStyle'
+export * from './styleUtils'
+export * from './withStyle'
```

Comment:
These export `*` are needed because of the [`isolatedModules` ](https://www.typescriptlang.org/tsconfig#isolatedModules) TS compiler flag.